libs/block-editor/src/lib/plugins/dot-bubble-menu.plugin.ts

```diff
@@ -373,4 +389,44 @@ export class DotBubbleMenuPluginView extends BubbleMenuView {
             this.tippyChangeTo?.hide();
         }
     }
+
+    private deleteNode() {
+        if (CustomNodeTypes.includes(this.selectionNode.type.name)) {
+            this.deleteSelectedCustomNodeType();
+        } else {
+            this.deleteSelectionNode();
+        }
+    }
+
+    private deleteSelectedCustomNodeType() {
+        const from = this.selection$FromPos.pos;
+        const to = from + 1;
+        this.editor.chain().deleteRange({ from, to }).blur().run();
```

Comment:
We should add a comment here because it's weird that the command does not work with `custom node`. It may be a problem with the command -[It's still a Draft](https://tiptap.dev/api/commands/delete-node)- but it can also be an issue with our `custom nodes`

The comment can be something like:
```
// TODO: Try to make the `deleteNode` command works with custom nodes.
```
or 

```
// TODO: Try the `deleteNode` command when a stable version is released

```